### ISSUE TITLE
DroneKit Tutorial

### ISSUE THREAD

**Issue by auturgy** [ArduPilot/ardupilot_wiki#5050]
[DroneKit Tutorial](https://ardupilot.org/dev/docs/droneapi-tutorial.html) - Embedded YouTube link no longer available

**Comment by Hwurzburg** [comment]
is this an acceptable replacement? https://www.youtube.com/watch?v=HD51_zvBn6U

**Comment by auturgy** [comment]
Not really. I'll have a chat to @peterbarker as we should probably provide some clarity/guidance more broadly regarding DroneKit.

**Comment by Hwurzburg** [comment]
@auturgy update? I think I updated the video...close?

**Comment by Hwurzburg** [comment]
No response since the 2023 'close?' question. Closing as stale.
